src/NpmUtilities.js

```diff
@@ -18,22 +18,29 @@ export default class NpmUtilities {
       stdio: ["ignore", "pipe", "pipe"],
     };
 
+    if (registry) {
+      opts.env = {npm_config_registry: registry};
+    }
+
     ChildProcessUtilities.spawn("npm", args, opts, callback);
   }
 
   @logger.logifySync()
-  static addDistTag(packageName, version, tag) {
-    ChildProcessUtilities.execSync(`npm dist-tag add ${packageName}@${version} ${tag}`);
+  static addDistTag(packageName, version, tag, registry) {
+    const opts = NpmUtilities.getTagOpts(registry);
+    ChildProcessUtilities.execSync(`npm dist-tag add ${packageName}@${version} ${tag}`, opts);
```

Comment:
I put it there because I thought that since NPM is handling the publish process it'd be a good enough fit there. By "making getTagOpts a function" do you mean a global distinct function?